### DISCUSSION TITLE
Flew scripts and styles

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#6941] [Help]
Everything worked fine, but today I found errors on the authorization page
![Снимок экрана 2023-07-06 155401](https://github.com/filamentphp/filament/assets/75489338/2c5e25a1-7378-49d4-9b92-3c2b848fdfa4)

**Comment by danharrin** [comment]
https://github.com/livewire/livewire/issues/242#issuecomment-948054946
